insights/client/connection.py

```diff
@@ -1100,7 +1115,12 @@ def checkin(self):
         url = self.inventory_url + "/hosts/checkin"
         logger.debug("Sending check-in request to %s with %s" % (url, canonical_facts))
         try:
-            response = self.post(url, headers={"Content-Type": "application/json"}, data=json.dumps(canonical_facts))
+            response = self.post(
+                url,
+                headers={"Content-Type": "application/json"},
+                data=json.dumps(canonical_facts),
+                log_response_text=False,
```

Comment:
ditto

Reply:
As noted above, checkin doesn't contain useful information, it's simply a full host dump.